### DISCUSSION TITLE
润学是个零和博弈

### DISCUSSION THREAD

**Discussion by xxxxxiaohao** [The-Run-Philosophy-Organization/run#181] [General]
润学最大的问题是，有多少人能成功润，并不取决于多少人动手开润，而是取决于目标国家的配额。以最受欢迎的美国来说，EB1/EB2/EB3是最多人的选择，然而每年给中国的配额是定死的，越多人走这条路，就会导致排期越长，并不会导致更多的人成功润掉。其他有限额的路线同理。所以我认为主流的路线全都是零和博弈，太多人润会导致严重的内卷，对谁都不利。

润学应该研究没有限额的路线。大家集思广益一下哪些国家是这样的政策？

我甚至觉得建议大家偷渡到美国然后非法的黑下，都比建议大家申请EB1/EB2/EB3来的有意义的多，因为黑下没有配额。

**Comment by xxxxxiaohao** [comment]
比如这个帖子就是一条好路：

https://www.uscardforum.com/t/topic/79255

**Comment by the-run-philosophy** [comment]
从来没有说润只能去美国。

**Reply by xxxxxiaohao** [reply]
> 从来没有说润只能去美国。

建议在这个项目里明确推荐哪些国家哪些项目是不限配额无限收的 这应该重点强调

**Comment by stackunderflow111** [comment]
很有意思
所以有没有寻找越南老婆的网站？

**Comment by yhzhang1** [comment]
这有点非黑即白了，排着也不影响你在美国工作不是

**Comment by yuchen001** [comment]
其实一个国家的人口缓慢持续增长，说明那个国家未来有希望
